src/SOS/extensions/hostcoreclr.cpp

```diff
@@ -70,8 +70,8 @@ namespace RuntimeHostingConstants
 {
     // This list is in probing order.
     constexpr RuntimeVersion SupportedHostRuntimeVersions[] = {
-        {7, 0},
         {6, 0},
+        {7, 0},
```

Comment:
Any reason to prefer 6 over 7

Reply:
6 is a LTS.  But 7.0 is the latest and has improvements so may be I'll remove this change.